.github/workflows/ci.yml

```diff
@@ -15,6 +15,7 @@ jobs:
   build:
     runs-on: ubuntu-latest
     strategy:
+      fail-fast: false
```

Comment:
why not fail fast? A comment in the code would be great here

Reply:
I will add more rationale to the commit message, would that be good enough?

As a general rule I'm trying to avoid non-functional parts of code as those tend to get outdated. An exception of this rule is the documentation of the externally consumable contract (e.g. API docs).